src/content/platform-integration/platform-channels.md

```diff
@@ -1349,7 +1315,16 @@ for iOS.
 }
 ```
 
-### Jumping to the UI thread in Android
+{% endtab %}
+
+{% endtabs %}
+
+:::note
+In release 2.10, the Task Queue API is only available on the `master` channel
```

Comment:
Two comments:
1) The 2.10 release was in October 2020. Do we still need to mention this?
2) Wherever possible, replace "master" with "main".

Reply:
This document is pretty old and from it what looks like, has only been updated here and there to mention a new feature. I'm betting we can delete this. I'll also go through and replace all 'master' references with 'main'.